book/src/proposals/durable-tx-nonces.md

```diff
@@ -0,0 +1,63 @@
+# Durable Transaction Nonces
+
+## Problem
+
+To prevent replay, Solana transactions contain a nonce field populated with a
+"recent" blockhash value. A transaction containing a blockhash that is too old
+(~2min as of this writing) is rejected by the network as invalid. Unfortunately
+certain use cases, such as custodial services, require more time to produce a
+signature for the transaction. A mechanism is needed to enable these potentially
+offline network participants.
+
+## A Naive Solution
```

Comment:
we don't need this section.  
I would cover the requirements
1. transaction needs to sign the nonce
2. nonce can only be used once, even if the key for the nonce account is leaked